src/app/util/client-api.h

```diff
@@ -152,16 +105,6 @@ uint16_t emberAfFillExternalManufacturerSpecificBuffer(uint8_t frameControl, chi
  */
 void emberAfSetExternalBuffer(uint8_t * buffer, uint16_t bufferLen, uint16_t * responseLengthPtr, EmberApsFrame * apsFramePtr);
```

Comment:
Why not pretty much everything in this file?

Reply:
Honestly?  I got tired of adding more stuff to this PR and needed to do other things.  Will do followups for more removals.